extensions/objects/templates/ImageWithRegionsEditor.js

```diff
@@ -31,14 +31,14 @@ oppia.directive('imageWithRegionsEditor', [
       },
       restrict: 'E',
       scope: {
-        value: '='
+        value: '@'
       },
       template: '<div ng-include="getTemplateUrl()"></div>',
       controller: [
         '$scope', '$element', '$uibModal',
         function($scope, $element, $uibModal) {
           $scope.alwaysEditable = true;
-
+          $scope.value = $scope.$parent.value;
```

Comment:
@seanlip and @kevinlee12: I tried setting the scope here to: 
```js
scope: {
   getValue: '&value'
}
```
and then `$scope.value = $scope.getValue()`. To check this method, I did a `console.log` : 
`console.log($scope.getValue)` to see if this is indeed a function, which it was and `$console.log($scope.value)` to see if the value was being passed. But `$scope.value` remains `undefined`.
On the other hand, `$scope.value = $scope.$parent.value` works.
Is there anything which I am missing here? Please correct me if I am doing something wrong.
I have been trying this for quite some time now. I have already gone through the documentation and other resources as suggested.

Thanks!

Reply:
Similarly for `GraphEditor.js`.

Reply:
Funny that this uses getTemplateUrl(). Seems a bit different from the other directives which reference templateUrl directly. /cc @vojtechjelinek do you know anything about this?

@apb7 I'm wondering if you included the attribute in your template? Usually all the stuff in the scope dict will need to be declared in the html tag.

If that doesn't work, send me minimal instructions on how to repro what you're seeing (from a fresh branch cut off of develop), and I'll see if I can poke around a bit with it tonight.

Reply:
Can you explain a bit more about how the HTML for this directive is actually generated, then? From my reading it comes from the ng-template and it's based on getTemplateUrl(). Can we get this directive to just use templateUrl instead like all the other ones?

Also, I think you'll want to ping @vojtechjelinek about this, too. I'm pretty sure he'll have more pointers, since he's worked a lot on this before.

Reply:
Regarding the questions, I am not too sure. I will discuss this with @vojtechjelinek once too.
Thanks!

Reply:
Hi, I think we first decided to skip all files in /extension (when we were directly referencing templates). Some /extensions folder use templateURL because I refactored them and as part of refactoring I directly referenced the templates. /cc @1995YogeshSharm do you know anything about this?

Reply:
Actually cc-ing @1995YogeshSharma . Yogesh, can you help here?